src/containers/Transactions/index.js

```diff
@@ -50,10 +54,9 @@ class Transaction extends Component {
     if (identifier && identifier !== hash) {
       actions.loadTransaction(identifier, rippledSocket);
     }
-  }
+  }, [identifier]);
```

Comment:
Do we also need to include `data` in this dependency array?

Reply:
No because this is what triggers the data request.  If we specified data it might create a loop.